ide/rocqide/document.ml

```diff
@@ -135,7 +135,10 @@ let is_empty = function
 
 let context d =
   let top, _, bot = to_lists d in
-  let pair _ x y = try Option.get x, y with Option.IsNone -> assert false in
+  let pair _ x y = match x with
+  | None -> assert false
```

Comment:
This is already the new behavior of get, so shouldn't we just drop the try with here?